### ISSUE TITLE
SmartCity-Accessibility Server Side Unit Tests #673 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#372]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 4.78 KB...]
Running smartcity.accessibility.mapmanagement.jxMapsFunctionalityTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 1, Time elapsed: 0.063 sec - in smartcity.accessibility.mapmanagement.jxMapsFunctionalityTest
Running smartcity.accessibility.mapmanagement.LocationTest

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 23, 2017
JxMaps license valid.
OpenJDK 64-Bit Server VM warning: INFO: os::commit_memory(0x00000001003e0000, 131072, 0) failed; error='Cannot allocate memory' (errno=12)
#
# There is insufficient memory for the Java Runtime Environment to continue.
# Native memory allocation (mmap) failed to map 131072 bytes for committing reserved memory.
# An error report file with more information is saved as:
# /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/hs_err_pid11034.log

Results :

Tests run: 1, Failures: 0, Errors: 0, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:25 min
[INFO] Finished at: 2017-05-23T19:27:14+00:00
[INFO] Final Memory: 35M/215M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: Execution default-test of goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test failed: The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side && /usr/lib/jvm/java-8-openjdk-amd64/jre/bin/java -javaagent:/var/lib/jenkins/.m2/repository/org/jacoco/org.jacoco.agent/0.7.8/org.jacoco.agent-0.7.8-runtime.jar=destfile=/var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/jacoco.exec -jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefirebooter8839350856055790076.jar /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire2492865336455778148tmp /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire/surefire_04765084107765423248tmp
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] d4fed8d3b45928f3bbecc94101ae787183cd04b8 - add npm install global gulp #354

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/673/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility Server Side Unit Tests' is failing!

Last 50 lines of build :
```
[...truncated 8.24 KB...]
Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.306 sec - in smartcity.accessibility.socialnetwork.ScoreTest
Running smartcity.accessibility.database.UserManagerTest
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 10.31 sec - in smartcity.accessibility.database.UserManagerTest
Running smartcity.accessibility.database.LocationManagerTest
Tests run: 9, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 6.633 sec - in smartcity.accessibility.database.LocationManagerTest
Running smartcity.accessibility.database.ReviewManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.202 sec - in smartcity.accessibility.database.ReviewManagerTest
Running smartcity.accessibility.database.UserProfileManagerTest
Tests run: 7, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.53 sec - in smartcity.accessibility.database.UserProfileManagerTest
Running smartcity.accessibility.search.NearbyPlacesTest
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.08 sec <<< FAILURE! - in smartcity.accessibility.search.NearbyPlacesTest
nearByPlacesTest(smartcity.accessibility.search.NearbyPlacesTest)  Time elapsed: 0.079 sec  <<< ERROR!
java.lang.IllegalStateException: HTML document isn't available.
	at smartcity.accessibility.search.NearbyPlacesTest.nearByPlacesTest(NearbyPlacesTest.java:34)

Running smartcity.accessibility.search.SearchTest
Tests run: 3, Failures: 0, Errors: 1, Skipped: 1, Time elapsed: 0.068 sec <<< FAILURE! - in smartcity.accessibility.search.SearchTest
test3(smartcity.accessibility.search.SearchTest)  Time elapsed: 0.02 sec  <<< ERROR!
java.lang.IllegalStateException: HTML document isn't available.
	at smartcity.accessibility.search.SearchTest.test3(SearchTest.java:81)


Results :

Tests in error: 
  NearbyPlacesTest.nearByPlacesTest:34 » IllegalState HTML document isn't availa...
  SearchTest.test3:81 » IllegalState HTML document isn't available.
  AdressSearchServiceTest.searchSuccess:34 » NestedServlet Request processing fa...

Tests run: 70, Failures: 0, Errors: 3, Skipped: 2

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 03:02 min
[INFO] Finished at: 2017-05-23T20:01:45+00:00
[INFO] Final Memory: 35M/211M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [yaelAmitay] d4fed8d3b45928f3bbecc94101ae787183cd04b8 - add npm install global gulp #354

- [yaelAmitay] e7e32e44ceef7abd6b0383ab3f43ffb5667da41a - change name weird bug



[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Unit%20Tests/674/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
